packages/vaadin-grid/src/vaadin-grid-dynamic-columns-mixin.js

```diff
@@ -13,6 +13,12 @@ import { timeOut } from '@polymer/polymer/lib/utils/async.js';
  */
 export const DynamicColumnsMixin = (superClass) =>
   class DynamicColumnsMixin extends superClass {
+    static get properties() {
+      return {
+        _columnTree: Object
```

Comment:
nit: JSDoc is missing, should probably be `@protected`.

Reply:
Marked as protected